core/rfb.js

```diff
@@ -718,6 +715,7 @@ RFB.prototype = {
                 Log.Error("Got data while disconnected");
                 break;
             case 'connected':
+                // eslint-disable-next-line no-constant-condition
```

Comment:
Yup. We could disable it globally. However, I think that's reasonable to assume that a condition on a constant should be an exception and not the norm. So it's disabled globaly and enabled only for this line.

Reply:
I don't like that we need to disable it in this case though. What is the eslint recommended way for an infinite loop?

Reply:
@juanjoDiaz, did you check if there was a better way that would make lint happy?

Reply:
there's apparently a `checkLoops` setting for exactly this. So I suggest turning on that option.

Reply:
Endless loops are dangerous. That's why there shouldn't be used unless strictly necessary.
That's why the `no-constant-condition` and `checkLoops` rule is enable by default. Disabling it will not indicate endless loop anywhere in the code.
That's why I think that using a comment is good to indicate that this particular endless loop is an exception and not the norm.

If you still disagree and prefer you to completely disable the rule. Let me know and I can do it.

Reply:
My vote is to disable this rule since I consider this particular pattern to be a common and useful one.

What does everyone else think?

Reply:
Agreed, +1 for disabling the rule